src/app.js

```diff
@@ -0,0 +1,75 @@
+import { CONSTANTS } from './modules/constants';
+import { addTodo } from './modules/todo';
+import { renderTodoList } from './modules/dom';
+
+function init() {
+  const root = document.getElementById('root');
+
+  const today = new Date().toLocaleDateString('ko-KR', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    weekday: 'long',
+  });
+
+  const header = document.createElement('header');
+  header.id = 'header';
+  header.textContent = CONSTANTS.HEADER;
+
+  const main = document.createElement('div');
+  main.id = 'main';
+
+  const title = document.createElement('h1');
+  title.id = 'title';
+  title.textContent = CONSTANTS.TITLE;
+
+  const dateElem = document.createElement('p');
+  dateElem.id = 'date';
+  dateElem.textContent = today;
+
+  const form = document.createElement('form');
+  form.id = 'todo-form';
+
+  const emoji = document.createElement('span');
+  emoji.textContent = CONSTANTS.EMOJI;
+
+  const input = document.createElement('input');
+  input.type = 'text';
+  input.placeholder = CONSTANTS.PLACEHOLDER;
+
+  const addBtn = document.createElement('button');
+  addBtn.type = 'submit';
+  addBtn.textContent = CONSTANTS.ADD;
+
+  form.appendChild(emoji);
+  form.appendChild(input);
+  form.appendChild(addBtn);
+
+  form.addEventListener('submit', (event) => {
+    event.preventDefault();
+
+    const text = input.value.trim();
+    if (!text) {
+      return;
+    }
```

Comment:
빈 문자열일 경우 바로 반환시키고 있는데, 사용자 경험을 위해서라면 alert문을 하나 넣어주는 것도 좋을 것 같아요. 아래와 같이 수정해보는 건 어떤가요?

```suggestion
    const text = input.value.trim();
    if (!text) {
      alert('할 일을 입력해 주세요!');
      return;
    }
```

Reply:
네 UX적인 부분에 대한 고려가 좀 부족했던 것 같네요 ㅜㅜ 추후 반영해서 수정하도록 하겠습니다!